Svc/ChronoTime/test/ut/ChronoTimeTestMain.cpp

```diff
@@ -0,0 +1,17 @@
+// ======================================================================
+// \title  ChronoTimeTestMain.cpp
+// \author mstarch
+// \brief  cpp file for ChronoTime component test main function
+// ======================================================================
+
+#include "ChronoTimeTester.hpp"
+
+TEST(Nominal, toDo) {
```

Comment:
Could you rename this `toDo` to something else like `TestBasicTime` ?

Reply:
Good catch!